akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/BasicDirectivesExamplesSpec.scala

```diff
@@ -183,7 +183,7 @@ class BasicDirectivesExamplesSpec extends RoutingSpec {
   }
   "withSettings-0" in compileOnlySpec {
     //#withSettings-0
-    val special = RoutingSettings(system).copy(fileIODispatcher = "special-io-dispatcher")
+    val special = RoutingSettingsImpl(system).copy(fileIODispatcher = "special-io-dispatcher")
```

Comment:
Whoops, sorry about forgetting that one. Yeah it should follow the same pattern as the others, Impl case class and java/scala  dsl classes